packages/utils/src/groupInMap.ts

```diff
@@ -50,3 +53,46 @@ export const groupInMap = <T, K, V>(
     }),
   )
 }
+
+const votesByKey$ = new Subject<{key: string}>()
```

Comment:
^^^ I think you forgot to remove the example from the source

Reply:
:facepalm: